### PR TITLE
Community Partner Highlight: African Institute of Biomedical Sciences and Technology (AiBST)

### LINKED ISSUE #91: Community Partner Highlight - AiBST
This issue tracks the drafting, review and approval for adding a new blog post to the Catalyst Project blog.

This issue is complete when: the copy of this page is approved and published on the website.

**Please edit and insert information enclosed by the <angled brackets> below**

Create a Google Doc in *Catalyst Project Team Drive > External Documentation > Blog posts > your_filename* to draft the copy for your new page, with a title of the form "[VERSION #] <your_title>" and append this title with "(ES)" if the draft is in Spanish.

Please consult the [Style Guide](https://docs.google.com/document/d/1qMSvTmqBuBow98tvxgNvsVZB9eo-pItWfGZPy8W_cok/edit?usp=sharing) for more information, including which metadata tags to choose from for your blog post.

Google doc for drafting copy:(https://docs.google.com/document/d/1BgAY9Hg1TruMsS3ysKX2_2iq33eIDf_jramZdBRAurc/edit)

**Roles**

- [x] Drafting content: @katiepratt
- [x] Sub-team review: @jnywong @Tajuddeen1 @LouWoodley 
- [x] Send to Author for sign off @katiepratt 
- [x] Any corrections made / final sign off @katiepratt 
- [x] Hand off to MD for translation @SLLDeC
- [x] Hand off to Jenny to draft website previews - 2 separate posts @jnywong 
- [x] Final approval of English post by Katie @katiepratt 
- [x] Final approval of Spanish post by Sabrina @SLLDeC

### PR DESCRIPTION
**Pull request by jnywong**
Closes #91 

English preview 👉 https://catalyst-website--98.org.readthedocs.build/en/98/blog/community-highlight-aibst-en.html
Spanish preview 👉 https://catalyst-website--98.org.readthedocs.build/en/98/blog/community-highlight-aibst-es.html

**TODO**

- [x] Finalise publication date

### CODE REVIEW

File: catalystbook/blog/community-highlight-aibst-es.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+author: Katie Pratt (CSCCE), Sabrina López (MetaDocencia)
+tags: Comunidades asociadas
+language: Español
+---
+
+# Comunidad Asociada Destacada - Instituto Africano de Ciencias Biomédicas y Tecnología (AiBST)
+
+*Esta publicación es parte de una serie que destaca a [las Comunidades asociadas del Proyecto Catalyst](../current-community-partners.md), quienes están utilizando la infraestructura en la nube del Proyecto Catalyst para avanzar en diversos proyectos en las biociencias. Nuestras Comunidades asociadas también juegan un papel vital en la configuración de nuestro modelo de gobernanza, ayudándonos a sostener, escalar y maximizar el impacto en América Latina, África y comunidades desatendidas en todo el mundo.*
+
+En esta publicación, Zedias Chikwambi comparte cómo la colaboración con el Proyecto Catalyst está impactando al Instituto Africano de Ciencias Biomédicas y Tecnología (AiBST, por African Institute of Biomedical Sciences and Technology) en Zimbabue.
```
Comment (SLLDeC) [inline catalystbook/blog/community-highlight-aibst-es.md:13]: When we mention the original name in English, it should be in italics.

```suggestion
En esta publicación, Zedias Chikwambi comparte cómo la colaboración con el Proyecto Catalyst está impactando al Instituto Africano de Ciencias Biomédicas y Tecnología (AiBST, por _African Institute of Biomedical Sciences and Technology_) en Zimbabue.
```

File: catalystbook/blog/community-highlight-aibst-es.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+:::
+
+**¿Podrías presentarte a nuestros lectores? Cuéntanos sobre tu organización/instituto/proyecto y sobre la investigación en la que está trabajando actualmente tu comunidad.**
+
+Soy Zedias Chikwambi del Instituto Africano de Ciencias Biomédicas y Tecnología (AiBST) en Zimbabue. Nuestra misión es descubrir y utilizar biomarcadores para guiar tratamientos personalizados. Aprovechamos una amplia gama de herramientas genómicas, incluidas bases de datos de secuenciación de nueva generación (NGS, por Next Generation Sequencing) , datos clínicos y plataformas computacionales avanzadas como los recursos en la nube del Proyecto Catalyst, para identificar y refinar biomarcadores. Estos biomarcadores se aplican en el desarrollo de herramientas de diagnóstico y detección de última generación. Los recursos computacionales a los que tenemos acceso han sido fundamentales para realizar análisis bioinformáticos exhaustivos en conjuntos de datos genómicos. Actualmente, nuestra comunidad de AiBST cuenta con cinco participantes activos, y estamos comprometidos en aumentar este número a través de iniciativas de formación locales y regionales.
```
Comment (SLLDeC) [inline catalystbook/blog/community-highlight-aibst-es.md:22]:
```suggestion
Soy Zedias Chikwambi del Instituto Africano de Ciencias Biomédicas y Tecnología (AiBST) en Zimbabue. Nuestra misión es descubrir y utilizar biomarcadores para guiar tratamientos personalizados. Aprovechamos una amplia gama de herramientas genómicas, incluidas bases de datos de secuenciación de nueva generación (NGS, por _Next Generation Sequencing_) , datos clínicos y plataformas computacionales avanzadas como los recursos en la nube del Proyecto Catalyst, para identificar y refinar biomarcadores. Estos biomarcadores se aplican en el desarrollo de herramientas de diagnóstico y detección de última generación. Los recursos computacionales a los que tenemos acceso han sido fundamentales para realizar análisis bioinformáticos exhaustivos en conjuntos de datos genómicos. Actualmente, nuestra comunidad de AiBST cuenta con cinco participantes activos, y estamos comprometidos en aumentar este número a través de iniciativas de formación locales y regionales.
```

File: catalystbook/blog/community-highlight-aibst-es.md [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+
+:::{pull-quote}
+A través del Proyecto Catalyst podemos llevar la interpretación de la información genómica a la cabecera del paciente.
```
Comment (SLLDeC) [inline catalystbook/blog/community-highlight-aibst-es.md:25]:
```suggestion
"A través del Proyecto Catalyst podemos llevar la interpretación de la información genómica a la cabecera del paciente."
```

File: catalystbook/blog/community-highlight-aibst-es.md [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+:::
+
+**¿Qué uso específico le están dando a la infraestructura en la nube? ¿Qué tipo de datos almacenan allí? ¿Qué paquetes de software utiliza tu comunidad?**
+
+Esta iniciativa está liderada por la Unidad de Genómica de AiBST, donde aprovechamos y celebramos la diversidad de variaciones genómicas entre individuos para entender mejor las respuestas a medicamentos e informar sobre terapias personalizadas. Nuestro trabajo incluye la generación y utilización de conjuntos de datos genómicos y clínicos, como datos transcriptómicos (tanto a nivel de célula única como a nivel de núcleos/células en conjunto), secuenciación completa del exoma (WES, por Whole Exome Sequencing), secuenciación completa del genoma (WGS, por Whole Genome Sequencing) y datos de genotipos obtenidos mediante plataformas de open array para polimorfismos de un solo nucleótido (SNPs, por Single Nucleotide Polymorphisms) y variaciones en el número de copias (CNVs, por Copy Number Variations). Utilizamos herramientas avanzadas como pipelines de identificación de variantes, R, Python y plataformas de visualización de datos para analizar e interpretar estos datos.
```
Comment (SLLDeC) [inline catalystbook/blog/community-highlight-aibst-es.md:30]:
```suggestion
Esta iniciativa está liderada por la Unidad de Genómica de AiBST, donde aprovechamos y celebramos la diversidad de variaciones genómicas entre individuos para entender mejor las respuestas a medicamentos e informar sobre terapias personalizadas. Nuestro trabajo incluye la generación y utilización de conjuntos de datos genómicos y clínicos, como datos transcriptómicos (tanto a nivel de célula única como a nivel de núcleos/células en conjunto), secuenciación completa del exoma (WES, por _Whole Exome Sequencing_), secuenciación completa del genoma (WGS, por _Whole Genome Sequencing_) y datos de genotipos obtenidos mediante plataformas de open array para polimorfismos de un solo nucleótido (SNPs, por _Single Nucleotide Polymorphisms_) y variaciones en el número de copias (CNVs, por _Copy Number Variations_). Utilizamos herramientas avanzadas como pipelines de identificación de variantes, R, Python y plataformas de visualización de datos para analizar e interpretar estos datos.
```

**Review by SLLDeC — CHANGES_REQUESTED** [review]
Hi @jnywong I just left some minor changes.

File: catalystbook/blog/community-highlight-aibst-en.md [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+:::
+
+**Could you introduce yourself to our readers? Tell us about your organisation/institute/project and the research the members of your community are working on right now.**
+
+I’m Zedias Chikwambi from the African Institute of Biomedical Sciences and Technology (AiBST) in Zimbabwe. Our mission is to discover and utilise biomarkers to guide personalised treatment. We leverage a wide range of genomic tools—including next generation sequencing (NGS) datasets, clinical data, and advanced computational platforms like the Catalyst Project’s cloud resources—to identify and refine biomarkers. These biomarkers are then applied in the development of cutting-edge screening and diagnostic tools. The computational resources available to us have been instrumental in performing comprehensive bioinformatics analyses on genomic datasets. Currently, our AiBST community consists of five active participants, and we are committed to expanding this number through local and regional training initiatives.
+
+:::{pull-quote}
+Through the Catalyst Project we are able to bring genomic information interpretation to patient bedsides."
```
Comment (jnywong) [inline catalystbook/blog/community-highlight-aibst-en.md:25]:
```suggestion
"Through the Catalyst Project we are able to bring genomic information interpretation to patient bedsides."
```

**Pull request merged**